server/README.md

```diff
@@ -16,6 +16,10 @@ generate that code by entering `go generate ./...` in the main folder of the pro
 Building the project works with `go build` in the main folder of the project. (Please keep in mind that you need to generate
 the assets first)
 
+## Or using Make
+
+You can easily build the server by using `make build` inside the `server` directory
```

Comment:
'_inside the server directory_' I would suggest to remote this part as the whole Readme is already associated with the server directory.

Reply:
Done